deep_water_level/src/deep_water_level/data.py

```diff
@@ -93,18 +93,45 @@ def __getitem__(self, index):
         return image, depth, image_path
 
 
-def get_transforms(crop_box=None):
+def get_transforms(crop_box=None, equalization:bool=True, is_trainig: bool=True):
     transforms_array = []
+
+    # TODO:
+    # - Randomize crop (within reason)
+    # - Color jitter
+    # - Make most effective use of dtypes
+
     if crop_box is not None:
         top, left, height, width = crop_box
         transforms_array.append(functools.partial(v2.functional.crop, top=top, left=left, height=height, width=width))
+
+    if equalization:
+        # Apply histogram equalization to each image
+        # IMPORTANT: This works with input in the range [0, 255]
+        transforms_array.append(v2.functional.equalize)
+
     transforms_array.extend(
         [
             v2.ToImage(),
             v2.ToDtype(torch.float32, scale=True),  # convert to float32 and normalize to 0, 1
+            # TODO(adamantivm) Enable normalization. Make it work with equalization, which requires
+            # moving equalization to later and including the necessary dtype conversions back and forth
+            # v2.Normalize(mean=[0.10391959, 0.1423446, 0.13618265], std=[0.05138122, 0.04945769, 0.045432]),
```

Comment:
It seems like doing equalization would undo some of the work of the normalization - maybe it wouldn't make sense to do both anyway?